packages/core/src/sync/transport.ts

```diff
@@ -45,7 +60,126 @@ export const cachedTransport = ({
       async request({ method, params }) {
         const body = { method, params };
 
+        // multicall
         if (
+          method === "eth_call" &&
+          params[0]?.data?.startsWith(MULTICALL_SELECTOR)
```

Comment:
Do we need to handle multiple version of multicall or anything here? Should just make sure it matches viem behavior.

Reply:
viem only uses `aggregate3()`

https://github.com/wevm/viem/blob/ea5a0ced1210873df75920b1c9d8afcbc8c6359e/src/actions/public/multicall.ts#L211-L223